tcp/src/tests.rs

```diff
@@ -215,9 +215,8 @@ fn peer_meta() {
 	println!("{}", result);
 
 	// contains random port, so just smoky comparing response length
-	assert_eq!(
-		if cfg!(windows) { 58 } else { 59 },
-		result.len()
+	assert!(
+		result.len() == 58 || result.len() == 59
```

Comment:
looks like this modification (58 for Windows) was only required for appveyor and not travis windows, I guess travis is using [visualstudio2017 toolchain](https://docs.travis-ci.com/user/reference/windows/#pre-installed-packages)

Reply:
Right, although afaict the port is just chosen randomly, so it's weird to assume that it's always going to be a 5-digit number.